lib/active_admin/views/components/status_tag.rb

```diff
@@ -57,9 +57,10 @@ def build(*args)
       protected
 
       def convert_to_boolean_status(status)
-        if status == 'true'
+        case status
+        when true, 'true'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.